services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationService.kt

```diff
@@ -128,15 +132,15 @@ open class ApplicationService(
         userId: String,
     ): Application {
         val application = getById(id)
-        val applicationBefore = application.toApplication()
+        val previous = application.toApplication()
```

Comment:
Maybe `previousApplication`? Calling it just `previous` makes me check what this is every time I see it.

Reply:
Renamed as suggested.